Apps/WebClient/test/functional/e2e/cypress/fixtures/encounterEnabledConfig.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
same comment that @sslaws has provided on my other PR, we need to find a way to load a json and be able to update some params in the code.